docs/src/layouts/BaseLayout.astro

```diff
@@ -44,8 +44,32 @@ const ogImage = `${Astro.site}og-banner-rocketsim.jpg`;
     <meta property="og:url" content={cannonicalUrl} />
     <meta name="twitter:card" content={ogImage} />
     <meta name="twitter:site" content="@rocketsim_app" />
+
+    <!-- Google Tag Manager -->
+    <script is:inline>
```

Comment:
@AvdLee This script is added to all pages, including the privacy and terms pages. Is that okay?